content/plugins/tobimori/inertia/plugin.txt

```diff
@@ -0,0 +1,67 @@
+Title: Inertia Adapter
+
+----
+
+Text:
+
+## What is Inertia.js?
+
+Inertia is a new approach to building classic server-driven web apps. It works similar to the backend architecture for [Kirby's Panel](https://getkirby.com/releases/3.6/fiber) - but built with an ecosystem around it that works with Vue, React and Svelte.
+
+Raw JSON data is returned from the server and rendered on the client-side. This allows you to build SPAs without having to build a separate API. Utilizing the server-side renderer, this makes perfect for websites that are heavily interactive but shouldn't suffer from SEO or performance issues.
+
+## Usage
+
+[Read the documentation](https://github.com/tobimori/kirby-inertia/blob/main/README.md#usage) to get started with Kirby and Inertia.
+
+## Support the project
+
+> This plugin is provided free of charge & published under the permissive MIT License. If you use it in a commercial project, please consider to [sponsor me on GitHub](https://github.com/sponsors/tobimori) to support further development and continued maintenance of Kirby SEO.
```

Comment:
Probably copy&paste mistake or do you really mean Kirby SEO on this plugin as well?